### ISSUE TITLE
[Request] "Insulated" Molecular Assembler

### ISSUE THREAD

**Issue by IMarvinTPA** [AppliedEnergistics/Applied-Energistics-2#2909]
#### Request Description
Edited from a bug-like issue.
The request is for a Molecular Assembler that does not transfer channels.  Thus allowing the ability to make a compact interface-assembler array that does not create a multiple-choice path problem.

#### Description
If I make a tower of interfaces and molecular assemblers such that A = Assembler and I = Interface and each layer is 
AIA
IAI
AIA
and make it 8 layers tall for 32 channels.

If there is a regular cable attached to the center assembler, when I reload the world, it only reloads with 24 channels and the center fiber has 8 from an unknown source.  However, if I have the dense cable up against the assembler or just air, it loads fine.  See screenshot of examples.
![2017-05-12_17 08 16](https://cloud.githubusercontent.com/assets/4486117/26017268/b98e37ca-3736-11e7-9232-8f7b6ba8d7b1.png)

#### Environment
FTB Beyond 1.6.1
- Minecraft Version:  1.10.2
- AE2 Version:  rv4 alpha 11
- Forge Version: 2281

**Comment by mindforger** [comment]
That is described in the FAQ devices only share 8 channels and when you stuff them together like this they suffer from racing conditions in finding paths try to pull up the wire on the outside or try not to stuff too much nlocks together

**Comment by IMarvinTPA** [comment]
Ah, I was under the impression that assemblers did not share channels because they did not use them.  I just did a quick test and found that they do.  Hmm.

**Comment by mindforger** [comment]
All ae2 devices act as cable, except the few like charger, growth accel and directed interface block have a few sides where they do not connect

**Comment by IMarvinTPA** [comment]
How about adding a new card called the "Terminator Card" that disables a device with it installed from continuing a network?

**Comment by leagris** [comment]
How about stating connected devices shares the same channel and pass-through all others up to 31?

Another option would be to allow facades and anchors on devices faces to block network linking with neighbor.

**Comment by CraftMinerKyle** [comment]
I thought channels are shared Horizontally, not Vertically.

**Comment by leagris** [comment]
Would love to have connected devices form one multiblock entity using 1 channel only.

**Comment by mindforger** [comment]
@IMarvinTPA though the idea beeing generally not bad, it would require to have EVERY block presenting a GUI, so this would involve a lot of work ... also all machines implement the standard cable interface, which makes them connect to cables in the first place and secondly would require special code to make em connect without sharing channels

@leagris hum i was under the impression it is stated on the official website ... oh wait it IS http://ae-mod.info/Channels/

also why should they share 32 channels? this would require to have a dense cable in every machine recipe as it is not inteded to have more than 8 channels in early game and presenting 32 on every connectable block would involve a cheaty way to build massive builds without making a single dense cable

**Comment by IMarvinTPA** [comment]
The lazy option for the Terminator card wouldn't require every block, just allow it for the ones that already accept a card.  I mostly only need it for Interface and Assembler blocks anyway.

I was looking at the code to figure out the nature of the pathing and the blocks favor attaching to cables, but other than that, there is a slight bias to Up, North and West (in that order.) 
I have not figure out how the system builds its master list of all things on the network and the order that it processes them to build up the connections.  But because when it makes a connection, it does it for both nodes involved and the order of the nodes seems to be the order they were created in, it leaves it a very hard to predict order.

Just for fun, I tried out all 6 directions of the bad build.  The structures that have the cabling on their Top, North or West sides work fine (although it may choose the middle connection vice the top one in the North pointing one.  The top set connect downward before connecting to the middle.

![2017-05-15_20 20 35](https://cloud.githubusercontent.com/assets/4486117/26084699/fb038ca6-39ab-11e7-8eb5-0430d3635f63.png)

I still don't know why it works when the center of the five is a dense cable (connecting to the surrounding cables) but it doesn't when it is a normal cable (blocked by anchors).  All versions work if there is no center cable at all.

My only other potential requested option is an "Insulated Molecular Assembler" that doesn't conduct channels.

Thanks for setting me straight on what is going on in general.  And I think I knew it one time, because my original design used interface bus panels on a single line of assemblers, and I know that doesn't conduct.  In that case, the dense cable also provided power to them.

**Comment by mindforger** [comment]
@IMarvinTPA can you edit the title of the issue to make it a [request] and modify the first post with your request for the "Insulated Molecular Assembler" ... i just fear all the posts might midlead someone for someone closing this discussion as it looks like beeing over :P

**Comment by yueh** [comment]
No
